event_registration.md

```diff
@@ -63,8 +63,7 @@ message QRCodePayload {
   uint32 version = 1;
   TraceLocation locationData = 2;
   CrowdNotifierData crowdNotifierData = 3;
-  // byte sequence of CWALocationData
-  bytes countryData = 4;
+  CWALocationData vendorData = 4;
```

Comment:
```suggestion
  // byte sequence of CWALocationData
  bytes vendorData = 4;
```

Reply:
ah yes, sorry I see.  
I tried to print the data and the bytes are CWALocationData, therefore I fixed it in my .proto file, but I just misread the comment, while wondering about the naming.